src/pattern-library/components/patterns/input/InputPage.js

```diff
@@ -49,7 +49,28 @@ export default function InputPage() {
           <Library.Example>
             <Library.Demo title="Basic Input" withSource>
               <div className="w-[350px]">
-                <Input placeholder="Placeholder..." />
+                <Input
+                  aria-label="Input example"
+                  placeholder="Placeholder..."
+                />
```

Comment:
Perhaps we should put some debug checks in the `<Input>` component to check for either a label-able ID or an `aria-label` at runtime.

Reply:
I thought about this! What should the behavior be if both are missing? Console warning? Something more serious?